core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -2363,6 +2363,11 @@ payment.country=Country
 payment.extras=Extra requirements
 payment.email.mobile=Email or mobile no.
 payment.altcoin.address=Altcoin address
+payment.altcoin.tradeInstantCheckbox=Trade instant
```

Comment:
Maybe it should be a little more descriptive like `Trade instant (within 1 hour) with this Altcoin` ⚡️?

Reply:
Ok, changed it.